app/helpers/application_notifications_helper.rb

```diff
@@ -0,0 +1,16 @@
+module ApplicationNotificationsHelper
+  # Returns the view path of the notification
+  #
+  # @param [Course::Notification|UserNotification] notification The target notification
+  # @return [String] The view path of the notification
+  def notification_view_path(notification)
```

Comment:
Assignment Branch Condition size for notification_view_path is too high. [17.29/15]